### ISSUE TITLE
[BUG] Steam parser lists only games that were installed at least once

### ISSUE THREAD

**Issue by rombat** [doZennn/steam-rom-manager#697]
- **Describe the bug**

Steam parser lists only games that were installed once ("Find artwork for installed titles only" is toggled off).
I use the latest Windows portable version

- **Steps to reproduce the behavior**

1. Create a Steam parser
2. Choose your Steam user account
3. Make sure that "Find artwork for installed titles only" is toggled off
4. Test the parser and check the list of found games. All the games listed have been installed once on your PC (and may still be installed, or not). Games that have never been installed are missing.

- **Expected behavior**

List all games, even the one that was never installed. It seems to me that in a previous version (can't remember which one) it was possible to list all Steam games, even the ones that were never installed.

**Comment by rombat** [comment]
I just tried the 2.5.17 and all games are listed, so there's probably a regression between 2.5.17 and 2.5.21

**Comment by kopiezzo** [comment]
Im having the same issue but 2.5.17 does not work for me anymore. Any fix?

**Comment by cbartondock** [comment]
Yeah this is a confirmed limitation of the current technique for parsing steam games -- previously they had to be in at least one category. We still haven't found a flawless way to do it.

**Comment by rombat** [comment]
Any news on this issue ? I'm still on 2.5.17 in the meantime :)

**Comment by kencinder** [comment]
Tested in 2.5.33 and this is still an issue, wouldn't exactly call it a bug but since it's an intended feature not working as intended...

**Comment by cbartondock** [comment]
The issue is that the previous technique required the games to be in at least one category, whereas the present technique just requires them to have been installed at least once.

I think the best we can do at the moment is to optimistically load from both, or provide it is a setting for the user and carefully specify the limitations of each.

I know of no technique at the moment to load games that a) have never been installed and b) are uncategorized. From the perspective of the local files that SRM reads, **those games don't exist**.

I'll work on adding the game finding strategy as a setting in the parser.

**Comment by cbartondock** [comment]
Actually, looking into this, the old method won't even work anymore since steam moved all of their category information. The limitation that it can only load in games that have been installed at least once is the best we've got for the moment. 

I'm going to close this for now; if someone can point me to local files that can reliably provide the user's steam games without any limitations I will implement parsing them.
